### PR TITLE
Use aiobotocore==2.0.0

### LINKED ISSUE #245: Needs to be updated to aiobotocore==2.0.0

### PR DESCRIPTION
**Pull request by mmaslowskicc**
Fixes #245.

### CODE REVIEW

File: tests/test_patches.py [hunk 1]
```diff
@@ -27,9 +27,9 @@
 
     # resources/action.py
     ServiceAction.__init__: {'b8b759abbe8fbfa9bad332b8ce8d30f55daf97f3'},
-    ServiceAction.__call__: {'3e84efcb2badef725b998e1af27a4a6f6f1aa2ed'},
-    BatchAction.__call__: {'8839f17467d16f98e4587fa750d74c9007dea134'},
-    WaiterAction.__call__: {'57d4f3f74e7e774e3aeb13630292fa9183a790f8'},
+    ServiceAction.__call__: {'f3cb58a5e36bf3355723c69ec244990180a2d5bc'},
```
Comment (mmaslowskicc) [inline tests/test_patches.py:30]: This seems caused only by changes in formatting, https://github.com/boto/boto3/commit/e2492c98c9c0f7787aa193a6709ff98ace09a5d8.

**Review by pablorf-dev — APPROVED** [review]

**Comment by darwinshameran** [comment]
👀

**Comment by rggjan** [comment]
2.0.1 is out

**Comment by terrycain** [comment]
Oh I completely missed this. Sorry guys, will bump to 2.0.1 tonight

**Comment by terrycain** [comment]
[9.3.0](https://pypi.org/project/aioboto3/) is out

**Pull request closed without merging**